openpype/hosts/nuke/plugins/create/create_write_render.py

```diff
@@ -47,6 +48,9 @@ def get_instance_attr_defs(self):
             self._get_reviewable_bool()
         ]
         if "farm_rendering" in self.instance_attributes:
+            render_farm_settings = RenderFarmSettings(
```

Comment:
Do you know that this method is called for each individual instance? So, when you have 10 instances in scene, the exact same code will be called 10 times (create `RenderFarmSettings` -> get project settings -> recalculate values -> create attributes). Maybe having a cache of `RenderFarmSettings` would help a lot, especially when you have access to project settings in initialization part.

Reply:
I see, then I suppose it would be smart to cache. But isn't this called only during creation? Way you explain it, it seems like it is called every time you refresh publisher, right?

Reply:
Yes, it's called on refresh for each instane and then on create for each created instance.

Reply:
done